### DISCUSSION TITLE
Похвастушки

### DISCUSSION THREAD

**Discussion by vvip-68** [vvip-68/GyverPanelWiFi#199] [Похвастушки]
Рассказывайте о своих достижениях, связанными с проектом, здесь

**Comment by sidi111** [comment]
Всем добрый день. Собрал матрицу 32х16. Большое спасибо автору за этот замечательный проект.
Прошивку использовал от [vvip-68](https://github.com/vvip-68/GyverPanelWiFi#chapter-0). Покупал 10 метров адресной ленты и резал на отдельные светодиоды.
Потом распаивал каждый на расстоянии 6 см по 16 светодиодов и наклеял на прозрачную основу.
Блок питания использовал на 40 ампер, с запасом.
По времени заняло 6 дней, но результат превзошел затраченное время и нервы.

[https://www.youtube.com/watch?v=rNJ2Lfy1nX8](url)

**Reply by Plohish67** [reply]
подскажи, а что за "прозрачная основа"?

**Comment by Krypt0nC0R3** [comment]
Хвастаюсь. Запилил программу, для управления Вашей лампой с помощью Алисы. Вот ссылка на репозиторий. Рассмотрю любые замечания и предложения.

https://github.com/Krypt0nC0R3/AliceMQTTHandler
